lib/elixir_sense/core/binding.ex

```diff
@@ -154,6 +154,34 @@ defmodule ElixirSense.Core.Binding do
     end
   end
 
+  # dependency injection
+  def do_expand(env, {:call, {:atom, Application}, fun, args}, stack)
+      when fun in ~w(get_env get_env! fetch_env fetch_env!)a do
+    try do
+      mod = apply(Application, fun, Enum.map(args, &elem(&1, 1)))
+
+      case mod do
+        {:ok, mod} ->
+          expand(env, {:atom, mod}, stack)
+
+        nil ->
```

Comment:
I'd remove this clause and let the next one return {:atom, nil}

Reply:
This is something that got me a bit confused and I forgot to ask. What's the difference between returning `nil`, `:none` or `{:atom, nil}` here?

Reply:
`{:atom, nil}` means that the expression is of type type `nil`
`nil` means that we are unable to tell the type; should be renamed to `:any`
`:none` means that the code is broken and no type satisfies the expressions

Reply:
Ok, following this definition I updated a bit the `case` branchs https://github.com/elixir-lsp/elixir_sense/pull/133/commits/06b479dceb15183dc8a26ccd1b5e2986e3b02539